programs/eosc/main.cpp

```diff
@@ -972,10 +977,11 @@ int main( int argc, char** argv ) {
          transaction_emplace_message(trx, config::eos_contract_name,
                                               vector<types::account_permission>{{sender,"active"}},
                                               "transfer", types::transfer{sender, recipient, amount, memo});
-         trx.expiration = info.head_block_time + tx_expiration; 
+         trx.expiration = info.head_block_time + fc::seconds(1800);
```

Comment:
Please add this as an argument to benchmark like it is for other command line options. Also use below.